volume-providers/vpc/messages/messages_en.go

```diff
@@ -191,6 +191,20 @@ var messagesEn = map[string]util.Message{
 		RC:          404,
 		Action:      "Run 'ibmcloud is volumes' to list available volumes in your account.",
 	},
+	"InvalidListVolumesLimit": util.Message{
+		Code:        "InvalidListVolumesLimit",
+		Description: "The value specified in the limit parameter of the list volume call is not valid.",
+		Type:        util.InvalidRequest,
+		RC:          400,
+		Action:      "Verify the limit parameter's value. The limit must be a positive number.",
```

Comment:
I think this limit should be in between 0-100, that we can also specify

Reply:
Fixed.